src/PrestaShopBundle/Form/Admin/Sell/Order/ChangeOrderCurrencyType.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
Update your license header generator (in your iDE) 😉 
```suggestion
 * 2007-2019 PrestaShop SA and Contributors
```